include/mxnet/engine.h

```diff
@@ -182,7 +182,7 @@ class MXNET_API Engine {
                          std::vector<VarHandle> const& mutable_vars,
                          FnProperty prop = FnProperty::kNormal,
                          int priority = 0,
-                         const char* opr_name = nullptr) = 0;
+                         const char* opr_name = nullptr, bool wait = false) = 0;
```

Comment:
what's wait? Why do you need it? Please document arguments

Reply:
wait is used to indicate  to the ExecuteOprBlock whether it is a waitforvar operation. it should not block the execution of the operator for WaitForVar.